breakerbp/failure_ratio.go

```diff
@@ -92,8 +119,10 @@ func (cb FailureRatioBreaker) runStatsProducer() {
 		case <-tick.C:
 			if cb.State() == gobreaker.StateOpen {
 				circuitBreakerGauge.Set(0)
+				breakerClosed.With(labels).Set(0)
 			} else {
 				circuitBreakerGauge.Set(1)
+				breakerClosed.With(labels).Set(1)
```

Comment:
These should be part of the `stateChanged()` method, so that the values are updated any time the `gobreaker.State` changes.